### PR TITLE
net-news/sfeed: add v1.6

### PR DESCRIPTION
**Pull request by N-R-K**

### CODE REVIEW

File: net-news/sfeed/sfeed-1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline net-news/sfeed/sfeed-1.6.ebuild:4]: 8 should work here.

**Review by juippis — APPROVED** [review]
You know, I would just install all of the themes unconditionally. It'd help to clean this ebuild by a whole lot.

File: net-news/sfeed/sfeed-1.6.ebuild (hunk 1 above)
Comment (N-R-K) [reply net-news/sfeed/sfeed-1.6.ebuild:4]: IIRC 8 was missing `savedconfig` (maybe that got changed or I'm misremembering ?).

**Comment by juippis** [comment]
Yes but this program is very small, compiling all of the themes adds an extra second?

**Comment by N-R-K** [comment]
> Yes but this program is very small, compiling all of the themes adds an extra second?

So install 4 different binaries? Or is there some misunderstanding going on?

Just so we're on the same page, the theme is selected at compile time and it's "hardcoded" into the binary (it's a bunch of C macros). So installing all themes would mean installing several binaries.

**Comment by juippis** [comment]
Oh okay I see, I thought I enabled all themes via use flag but it seems I had `-ncurses` set so it allows it. Okay, thanks for explaining :)

**Pull request closed without merging**